### PR TITLE
Added sign-up page

### PR DESCRIPTION
**Pull request by srollins01**
Created sign-up route and page with Firebase authentication.

### CODE REVIEW

File: friendster-frontend/vite-project/src/components/Login.jsx [hunk 1]
```diff
@@ -23,6 +23,11 @@ const Login = () => {
       .then((idToken) => {
         // Set the ID token to local storage
         localStorage.setItem("userToken", idToken);
+        // Set expiration time (e.g., 1 hour from now)
+        const expirationTime = new Date().getTime() + 30*10 * 1000;//
```
Comment (srollins01) [inline friendster-frontend/vite-project/src/components/Login.jsx:27]: It looks like the token expiration time is set to 5 minutes. I believe you said you wanted to set it to 30 minutes?

**Comment by srollins01** [comment]
@amante739 Thanks for fixing the sign-up issues. I've rebased our commits with the master branch and made some adjustments + left feedback. Please have a look when you have time.

**Review by Paulette-Zaldivar-Flores — CHANGES_REQUESTED** [review]
@srollins01 @amante739 I tested the sign up function but could not get past this screen telling me there is a problem with sign up and it does not tell me what the problem is. I did an npm beforehand to make sure axios was installed. Is there anything else I should be doing when testing this?
Screenshot 2567-04-24 at 16 49 14

**Comment by Paulette-Zaldivar-Flores** [comment]
@srollins01 @amante739 I tested the sign up feature and get this  500 error coming from axios. Not sure what the problem since I did npm i on both frontend and backend. Check it out.

https://github.com/Paulette-Zaldivar-Flores/Friendster/assets/96970580/0a83044b-5e97-47b5-a6bb-fae69ce649f9

**Pull request merged**